### PR TITLE
RavenDB-14532 allow to specify scale in timeseries queries

### PR DESCRIPTION
**Pull request by aviv86**
https://issues.hibernatingrhinos.com/issue/RavenDB-14532

### CODE REVIEW

File: src/Raven.Server/Documents/Queries/AST/Query.cs [hunk 1]
```diff
@@ -812,7 +812,9 @@ public IEnumerable<object> GetFinalValues()
                     throw new ArgumentOutOfRangeException("Unknown aggregation operation: " + Aggregation);
             }
 
-            return _values.Cast<object>();
+            return scale.HasValue 
+                ? _values.Select(v => (object)(v * scale.Value))
```
Comment (ppekrol) [inline src/Raven.Server/Documents/Queries/AST/Query.cs]: Can we avoid LINQ here and use local static method instead (for both of the cases) ?

Comment (karmeli87) [reply src/Raven.Server/Documents/Queries/AST/Query.cs]: Or can we scale it while we are adding the results?

Comment (ayende) [reply src/Raven.Server/Documents/Queries/AST/Query.cs]: Scaling while adding is how I would like to go about it.

Comment (aviv86) [reply src/Raven.Server/Documents/Queries/AST/Query.cs]: done

**Review by ppekrol — APPROVED** [review]

**Review by ayende — APPROVED** [review]

**Pull request merged**